inst/qml/TTestIndependentSamples.qml

```diff
@@ -77,6 +77,10 @@ Form
 			name: "descriptivesPlots";		label: qsTr("Descriptives plots")
 			CIField { name: "descriptivesPlotsConfidenceInterval"; label: qsTr("Confidence interval") }
 		}
+		CheckBox
```

Comment:
Did you manage to change the indenting of `qml` to tabs in Qt? It seems that the indenting is wrong here.

Reply:
I will have a look again.